docs/source/quickstart/model_config.md

```diff
@@ -4,7 +4,15 @@ The `ModelConfig` is created from a YAML file and defines the scoring layers of
 
 Currently, ml4ir supports a dense neural network architecture (multi layer perceptron like). Users can define the type of scoring architecture using the `architecture_key`. The layers of the neural network can be defined as a list of configurations using the `layers` attribute. For each layer, define the `type` of tensorflow-keras layer. Then for each layer, we can specify arguments to be passed to the instantiation of the layer. Finally, for each layer, we can specify a name using the `name` attribute.
 
-Below you can see an example model config YAML using a DNN architecture to stack a bunch of dense layers with ReLU activation layers. Additionally, there are also a few dropout layers for regularization in between.
+This file is also used to define the optimizer and the learning rate schedule. The current supported optimizers are: `adam`, `adagrad`, `nadam`, `sgd`, `rms_prop`. Each of these optimizers need so set the following hyper-parameter: `gradient_clip_value`. 
+The current supported learning rate schedules are: `exponential` and `cyclic`. 
```

Comment:
@mohazahran Should we give a name to the default learning rate schedule? Perhaps `uniform`?

Reply:
you may want to consider `constant`.